test/services/EnglishUIBuilderSpec.scala

```diff
@@ -499,7 +472,7 @@ class UIBuilderSpec extends BaseSpec with ProcessJson {
       val p = uiBuilder.buildPage("/start", Seq(SubSectionCallout(headingPhrase, Seq.empty, false),
                                                 tableRowGroup))
       p.components match {
-        case Seq(Table(Text(headingPhrase,_), tableHeading, _)) => succeed
+        case Seq(Table(Text(headingPhrase), tableHeading, _)) => succeed
```

Comment:
I'm not certain how this is working. I can't see an apply method that takes a Phrase argument and there is no toString on Phrase.

Reply:
its a bit deceptive, the headingPhrase in the match (Table(Text(headingPhrase) ... is a filled in arg to the match not the val defined earlier. recoded to be less deceptive and test further

Reply:
Thanks for the explanation